docs/ReferenceManyField.md

```diff
@@ -185,8 +186,35 @@ You can filter the query used to populate the possible values. Use the `filter`
    ...
 </ReferenceManyField>
 ```
+
+{% endraw %}
+
+## Filter Inputs
+
+<video controls autoplay playsinline muted loop>
+  <source src="./img/ReferenceManyFieldFilterInput.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>
+
+You can add an array of filter inputs to the `ReferenceManyField` using [the `FilterForm` component](./FilterForm.md):
```

Comment:
You should be able to add the `<Filter>` component directly and have both the button and the form like for the list